src/Symfony/Component/Mime/Tests/EmailTest.php

```diff
@@ -238,6 +238,12 @@ public function testGenerateBodyThrowsWhenEmptyBody()
         (new Email())->getBody();
     }
 
+    public function testGenerateBodyDoesNotThrowWhenAttachmentsPresent()
+    {
+        $body = (new Email())->attachFromPath(__FILE__)->getBody();
+        $this->assertNotEmpty($body);
```

Comment:
I would check the structure of the email as you mentioned that it is important for you.

Reply:
Great idea, new test with this assertion moved to `testGenerateBody()`